test/com/google/javascript/jscomp/PolymerBehaviorExtractorTest.java

```diff
@@ -28,8 +27,6 @@
  */
 public class PolymerBehaviorExtractorTest extends CompilerTypeTestCase {
 
-  private static final Joiner LINE_JOINER = Joiner.on('\n');
```

Comment:
This is used, but we could rename the one in CompilerTypeTestCase and make it protected.